cmd/reviewdog.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/stevemcquaid/mcq/pkg/commands"
+)
+
+var ReviewDogCmd = &cobra.Command{
+	Use:   "reviewdog",
+	Short: "-> reviewdog",
+	Long:  `Runs reviewdog`,
+	Run: func(cmd *cobra.Command, args []string) {
+		_ = commands.ReviewDog(PRNumFlag, SuggestFlag)
+	},
+}
+
+var (
+	PRNumFlag       int
```

Comment:
🚫 **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File is not `gofmt`-ed with `-s` (gofmt)